go.mod

```diff
@@ -7,6 +7,8 @@ replace google.golang.org/grpc => google.golang.org/grpc v1.29.0
 
 replace github.com/opencontainers/go-digest => github.com/opencontainers/go-digest v1.0.1-0.20220411205349-bde1400a84be
 
+replace github.com/open-component-model/ocm-e2e-framework => ../ocm-e2e-framework
```

Comment:
Remove this!

Added this because adding a component now requires https. I will have to fix that in the framework.